src/state/lists/reducer.ts

```diff
@@ -111,30 +110,4 @@ export default createReducer(initialState, (builder) =>
         pendingUpdate: null,
       }
     })
-    .addCase(updateVersion, (state) => {
```

Comment:
@vm is this code intended to run every session? (that's my understanding of how `updateVersion` works)

i want to move it to the initial migration to version 0 of the new system - see `migrations.ts` - and i'm wondering if lists will break if it doesn't run on every session.